### PR TITLE
feat(docs): Update documentation

### PR DESCRIPTION
**Pull request by PatStiles**

### CODE REVIEW

**Comment by entropidelic** [comment]
the installation of zkRust worked ok with the script, but if the sp1 and risc0 toolchains are not installed, the binary does not work

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ zkRust supports generating proofs for executable scripts. Specifically, zkRust s
 First make sure [Rust](https://www.rust-lang.org/tools/install) is installed on your machine. Then install the zkVM toolchains from [risc0](https://github.com/risc0/risc0) and [sp1](https://github.com/succinctlabs/sp1) by running:
 
 ```sh
-make install
+curl -L https://raw.githubusercontent.com/yetanotherco/zkRust/main/install_zkrust.sh | bash
```
Comment (entropidelic) [inline README.md:12]: this should not be replaced, we need a guide for local development. Keep the `make install` and make another section for installing directly

Comment (entropidelic) [reply README.md:12]: also, the downloaded binary does not work without the toolchains, so we may have to give guidelines to install them or add their installation to the script

Comment (PatStiles) [reply README.md:17]: Addressed!

**Pull request merged**